### PR TITLE
Add dev site to readme

### PR DESCRIPTION
**Pull request by sambehrens**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,9 +1,16 @@
...
 [![Build Status](https://travis-ci.org/dog-house-development/poodl.svg?branch=dev)](https://travis-ci.org/dog-house-development/poodl)
 
-_Welcome to Poodl!_ Find our application at [https://poodl.herokuapp.com/](https://poodl.herokuapp.com/ "poodl"). Please read [our wiki](https://github.com/dog-house-development/poodl/wiki "poodl wiki") to learn more.
+_Welcome to Poodl!_ Find our application at [https://poodl.herokuapp.com/](https://poodl.herokuapp.com/ 'poodl'). Please read [our wiki](https://github.com/dog-house-development/poodl/wiki 'poodl wiki') to learn more.
+
+### Sites
+
+**dev:** [poodl-dev.herokuapp.com](http://poodl-dev.herokuapp.com/)
+
+**prod:** [poodl.herokuapp.com](http://poodl-dev.herokuapp.com/)
```
Comment (karlmolina) [inline README.md]: write out production

File: README.md [hunk 2]
```diff
@@ -1,9 +1,16 @@
 # Poodl
+
 [![Build Status](https://travis-ci.org/dog-house-development/poodl.svg?branch=dev)](https://travis-ci.org/dog-house-development/poodl)
 
-_Welcome to Poodl!_ Find our application at [https://poodl.herokuapp.com/](https://poodl.herokuapp.com/ "poodl"). Please read [our wiki](https://github.com/dog-house-development/poodl/wiki "poodl wiki") to learn more.
+_Welcome to Poodl!_ Find our application at [https://poodl.herokuapp.com/](https://poodl.herokuapp.com/ 'poodl'). Please read [our wiki](https://github.com/dog-house-development/poodl/wiki 'poodl wiki') to learn more.
+
+### Sites
+
+**dev:** [poodl-dev.herokuapp.com](http://poodl-dev.herokuapp.com/)
```
Comment (karlmolina) [inline README.md]: write out development

**Review by karlmolina — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (sambehrens) [reply README.md]: Done

File: README.md (hunk 2 above)
Comment (sambehrens) [reply README.md]: Done

**Review by karlmolina — APPROVED** [review]

**Pull request merged**